### ISSUE TITLE
Version check for polyfills

### ISSUE THREAD

**Issue by SemenchenkoVitaliy** [metarhia/common#253]
Sometimes we need to implement polyfills for older Node.js versions and for this we have to check version manually like:
```js
const vArr = process.versions.node.split('.').map(num => parseInt(num));
if (vArr[0] > 8 || (vArr[0] === 8 && vArr[1] >= 2)) {
  ...
}
```
I propose to implement function that will check if process' version is older than required.
```js
if (common.nodeVersionCompatible('8.2')) {
  ...
}
```
or
```js
if (common.v8VersionCompatible('8.2')) {
  ...
}
```
WDYT?
/cc @lundibundi @nechaido @belochub @tshemsedinov

**Comment by nechaido** [comment]
AFAIK we need to check that the current version is equal or higher than the specified version, not compatible.
Example: we only use mkdip polyfill for <10.12.0.
 
| Version | Polyfill | v10.12.0 compatible |
|---------|---------|------------|
| 8.0.0   | :heavy_check_mark:  | :negative_squared_cross_mark:  |
| 10.0.0  | :heavy_check_mark:  | :negative_squared_cross_mark:  |
| 10.12.0 | :negative_squared_cross_mark:  | :heavy_check_mark:  |
| 11.0.0  | :negative_squared_cross_mark:  | :negative_squared_cross_mark:  |

**Comment by SemenchenkoVitaliy** [comment]
@nechaido Agree, function name should be changed. What about `verifyVersion` or `checkVersion`? Can as well make `compareVersions`.
